nixos/modules/system/boot/initrd-ssh.nix

```diff
@@ -132,9 +132,9 @@ in
       Ciphers ${concatStringsSep "," sshdCfg.ciphers}
       MACs ${concatStringsSep "," sshdCfg.macs}
 
-      LogLevel ${sshdCfg.logLevel}
+      LogLevel ${sshdCfg.options.LogLevel}
```

Comment:
shouldn't it be `settings` instead of `options` ? other than that LGTM. It's a bit weird that the sshd_config writing logic is duplicated since it leads to this kind of issues